### ISSUE TITLE
LongAP One

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/hm-pyhelper#180]
AMD processor
Nfuse USB sx1303 concentrator
Intel WiFi
Removable 6 pin ECC on gpio
Button and 3 LEDs
SSD over msata (looks like 64gb)
Seems to also have an RTC as there is a coin cell battery

![20220704_193017](https://user-images.githubusercontent.com/3359418/177216814-4fdcf669-fb9e-4855-a043-039e158e2c33.jpg)
![20220704_193013](https://user-images.githubusercontent.com/3359418/177216818-af9a3b4f-b080-4088-bf90-984147b2b677.jpg)
![20220704_193022](https://user-images.githubusercontent.com/3359418/177216821-32ac7974-46bd-4557-ba5c-222a3ff928c7.jpg)
![20220704_193147](https://user-images.githubusercontent.com/3359418/177216824-6ac42da4-0806-40c5-b00e-d34b3ac84550.jpg)
![20220704_193154](https://user-images.githubusercontent.com/3359418/177216828-6181448c-d76d-4611-9cd8-d8eea5c84d1a.jpg)
![20220704_193157](https://user-images.githubusercontent.com/3359418/177216833-00a95935-b834-4a74-9e20-46d72f885b35.jpg)
![20220704_193159](https://user-images.githubusercontent.com/3359418/177216837-a0a6da7d-1815-4950-958f-78b5127145e5.jpg)
![20220704_193238](https://user-images.githubusercontent.com/3359418/177216842-d43b0cda-6c06-4fb2-a93d-392b253aafb4.jpg)
![20220704_193256](https://user-images.githubusercontent.com/3359418/177216845-0ddfda3b-695b-4bc0-9587-036bce6c253b.jpg)
![20220704_193328](https://user-images.githubusercontent.com/3359418/177216850-769b0fe5-8c29-49ed-bc51-468ef70319e2.jpg)
![20220704_193338](https://user-images.githubusercontent.com/3359418/177216852-5f73ec60-e148-4d2f-b827-216533def3ee.jpg)
![20220704_193348](https://user-images.githubusercontent.com/3359418/177216854-c78a4860-b8de-4f17-a3b6-e09dc5519cbc.jpg)

**Comment by shawaj** [comment]
Chip PDF 
[g-series-soc-product-brief.pdf](https://github.com/NebraLtd/hm-pyhelper/files/9042102/g-series-soc-product-brief.pdf)

**Comment by shawaj** [comment]
This is the unit https://status.longap.com/5AE9B8

**Comment by shawaj** [comment]
GPIO stuff 

https://github.com/pcengines/apu_gpio_lib

https://pcengines.ch/howto.htm#gpio

**Comment by shawaj** [comment]
Manual 
[LongAP-One-Manual-v1.0.pdf](https://github.com/NebraLtd/hm-pyhelper/files/9042125/LongAP-One-Manual-v1.0.pdf)

**Comment by shawaj** [comment]
Looks like all LEDs are green... The manual above shows the functionality.

![longap-helium-hotspot-miner](https://user-images.githubusercontent.com/3359418/177219583-8b661ca2-91c4-404c-ada3-c7986cae0c71.png)
![longap-helium-hotspot-miner-2](https://user-images.githubusercontent.com/3359418/177219584-460852fc-5f68-4670-8c82-4c0f566e0579.png)

**Comment by shawaj** [comment]
@MuratUrsavas I wonder if this bit of blue tape sticking out is something to do with what you mentioned and needing to stop those signals....

![image](https://user-images.githubusercontent.com/3359418/179206278-a58f03ae-0ee1-48a3-9de9-ef84c8c88eb0.png)

**Comment by MuratUrsavas** [comment]
> @MuratUrsavas I wonder if this bit of blue tape sticking out is something to do with what you mentioned and needing to stop those signals....

@shawaj That's very probable. In the concentrator design those pins are for SX1261 and should be not connected to the mPCIe pins. Looks like the somehow have them connected and got a quick solution for that :smile:

**Comment by MuratUrsavas** [comment]
Looks like they have also wired SWD lines of the STM32 MCU to the mPCIe interface. Might create additional conflict points according to the design and used base hardware (PCI lane count)

**Comment by shawaj** [comment]
Yeah we aren't using that one so pinout will not be the same as that. But just thought it was interesting in any case

**Comment by MuratUrsavas** [comment]
And it's not a reliable way to interfere the signals :smile:
